### ISSUE TITLE
Not able to find classes after changing the version of the library to 2.0.0

### ISSUE THREAD

**Issue by Shajeel-Afzal** [wasabeef/recyclerview-animators#48]
I changed the version of the library to:

```
compile 'jp.wasabeef:recyclerview-animators:2.0.0'
```

And now it is not able to find any Animator, for example proejct is giving me error for:

```
        FadeInUpAnimator animator = new FadeInUpAnimator();
        animator.setAddDuration(200);
```

What might be wrong?

**Comment by wasabeef** [comment]
@Shajeel-Afzal 

what's your use version?

If you use the `recyclerview-animator:2.0.0` you need to use the `recyclerview-v7:23.1.0`.
```
compile "com.android.support:recyclerview-v7:23.1.0"
```

**Comment by Shajeel-Afzal** [comment]
Yes i am using the same version of RecyclerView (23.1.0). But issues seems to be with import.

**Comment by wasabeef** [comment]
@Shajeel-Afzal 
What kind of errors?

**Comment by Shajeel-Afzal** [comment]
@wasabeef it is not able to find any class from the library. for example `FadeInUpAnimator` it is not importing it.

**Comment by Shajeel-Afzal** [comment]
This is also happening in case of https://github.com/prolificinteractive/material-calendarview, however i can use the class in the XML but i am not able to import it in the Code. I have tried other libraries like `Retrofit` to include in the code and they are working fine. What is wrong with some of the Github projects?

**Comment by Shajeel-Afzal** [comment]
Well, i just observed that i can import once i run the app in case of `MaterialCalendarView`, i think the same will also work for `recyclerview-animators`.

**Comment by oktayayar** [comment]
I have same issue. I downloaded jar file and imported my project and Animation classes are not recognized by the compiler and they cannot be imported

**Comment by wasabeef** [comment]
@oktayayar 

whats jar version?

**Comment by oktayayar** [comment]
@wasabeef Version 2.0.0. I downloaded from maven repository given in description. However, I solved this issue by importing source codes as library to my project.
